rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -576,10 +576,23 @@ static NEVER_INLINE s32 savedata_op(ppu_thread& ppu, u32 operation, u32 version,
 		{
 			cellSaveData.warning("savedata_op(): funcStat returned result=%d.", result->result);
 
+			//Skip and error
+			if (result->result >= CELL_SAVEDATA_CBRESULT_OK_LAST_NOCONFIRM || result->result < CELL_SAVEDATA_CBRESULT_ERR_INVALID)
+			{
+				// ****** sysutil savedata parameter error : 22 ******
+				return CELL_SAVEDATA_ERROR_PARAM;
+			}
+
 			if (result->result < CELL_SAVEDATA_CBRESULT_OK_NEXT)
 			{
 				return CELL_SAVEDATA_ERROR_CBRESULT;
 			}
+
+			//Skip and return without error
+			if (result->result == CELL_SAVEDATA_CBRESULT_OK_LAST)
```

Comment:
Need to remove this part (see #4397)

Reply:
Hmm that's not correct, I'll take a look at the game but from what I see in disassembly it terminates without calling any other callback.

Reply:
Reversed Battle Princess of Arcadias, the launch race condition has nothing to do with save functions, no reason to keep the hack in.